persistence-api/src/main/java/io/stargate/db/schema/SchemaBuilderImpl.java

```diff
@@ -51,27 +51,27 @@
     })
 public class SchemaBuilderImpl {
 
-  private Set<Keyspace> keyspaces = new LinkedHashSet<>();
-  private Set<Table> tables = new LinkedHashSet<>();
-  private Set<Column> columns = new LinkedHashSet<>();
-  private Set<Column> materializedViewColumns = new LinkedHashSet<>();
-  private Set<Index> indexes = new LinkedHashSet<>();
+  private final Set<Keyspace> keyspaces = new LinkedHashSet<>();
+  private final Set<Table> tables = new LinkedHashSet<>();
+  private final Set<Column> columns = new LinkedHashSet<>();
+  private final Set<Column> materializedViewColumns = new LinkedHashSet<>();
+  private final Set<Index> indexes = new LinkedHashSet<>();
   private String keyspaceName;
   private String tableName;
   private String materializedViewName;
   private Column secondaryIndexColumn;
   private String secondaryIndexName;
-  private Optional<Consumer<Schema>> callback;
+  private final Optional<Consumer<Schema>> callback;
   private String udtTypeName;
-  private Set<UserDefinedType> udts = new LinkedHashSet<>();
+  private final Set<UserDefinedType> udts = new LinkedHashSet<>();
   private boolean indexKeys;
   private boolean indexValues;
   private boolean indexEntries;
   private boolean indexFull;
   private String indexClass;
   private Map<String, String> indexOptions = new HashMap<>();
-  private List<Column> fromColumns = new ArrayList<>();
-  private List<Column> toColumns = new ArrayList<>();
+  private final List<Column> fromColumns = new ArrayList<>();
+  private final List<Column> toColumns = new ArrayList<>();
   private Map<String, String> replication = Collections.emptyMap();
   private Optional<Boolean> durableWrites = Optional.empty();
```

Comment:
this seems mixed now, maybe revert?

Reply:
Or reorganize so the `final' are together?

Reply:
Reverted.